spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -94,6 +94,9 @@ spring:
         dependency-fetch:
           enabled: false
         dependencies:
+          spring-cloud-dataflow-server:
+            name: Spring Cloud Dataflow
+            version: "@project.version@"
           spring-cloud-dataflow-dashboard:
```

Comment:
Where these entries in PRO required since they are the same here (OSS)? https://github.com/pivotal/scdf-pro/blob/1483cf2b965dbc4e0d7be3da3a8ada2aff6473d3/scdf-pro-server/src/main/resources/application.yml#L19-L30

Reply:
Yes. The properties from the 2 files won't be merged.